.github/workflows/release.yaml

```
@@ -41,8 +42,13 @@ jobs:
     - name: Install pypa/build
       run: python3 -m pip install build --user
     
-    - name: Build a binary wheel and a source tarball
-      run: python3 -m build
+    - name: Install and configure Poetry
+      uses: snok/install-poetry@v1
```

```suggestion
      uses: snok/install-poetry@76e04a911780d5b312d89783f7b1cd627778900a
```
The commit hash is required instead of the version number.

---

solved